### PR TITLE
fix(observable): fix descriptor having both value and get and set

### PR DESCRIPTION
**Pull request by Gheoan**
Related: https://github.com/aurelia/binding/issues/511.

### CODE REVIEW

**Comment by Gheoan** [comment]
Added a new test and changed old ones.

File: test/decorator-observable.spec.js [hunk 1]
```diff
@@ -70,45 +70,76 @@ describe('observable decorator', () => {
     expect(instance.value).toEqual(newValue);
   });
 
-  it('should work with decorators function', () => {
-    const instance = new (decorators(observable('value'))
-      .on(class {
+  describe('es2015 with decorators function', () => {
+    xit('should work when decorating property', () => {
```
Comment (Gheoan) [inline test/decorator-observable.spec.js]: This test is pending https://github.com/aurelia/metadata/pull/44.

Comment (Gheoan) [inline test/decorator-observable.spec.js]: No longer pending.

**Comment by EisenbergEffect** [comment]
@jdanyow Assigned back to you since @Gheoan appears to have added the test :)

**Pull request merged**